SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "EngineClasses/SpatialLoadBalanceEnforcer.h"
+#include "EngineClasses/SpatialVirtualWorkerTranslator.h"
+#include "EngineClasses/SpatialNetDriver.h"
+#include "Interop/Connection/SpatialWorkerConnection.h"
+#include "Interop/SpatialSender.h"
+#include "Interop/SpatialStaticComponentView.h"
+#include "Schema/AuthorityIntent.h"
+#include "SpatialCommonTypes.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialLoadBalanceEnforcer);
+
+using namespace SpatialGDK;
+
+USpatialLoadBalanceEnforcer::USpatialLoadBalanceEnforcer(const FObjectInitializer& ObjectInitializer)
+	: Super(ObjectInitializer)
+	, NetDriver(nullptr)
+	, Sender(nullptr)
+	, VirtualWorkerTranslator(nullptr)
+{
+}
+
+void USpatialLoadBalanceEnforcer::Init(USpatialNetDriver* InNetDriver,
+	USpatialSender* InUpatialSender,
+	SpatialVirtualWorkerTranslator* InVirtualWorkerTranslator)
+{
+	NetDriver = InNetDriver;
```

Comment:
What is the net driver needed for? Can we pull those dependencies out?

Reply:
Just StaticComponentView

Reply:
It looks like it's mostly static component view.